### DISCUSSION TITLE
Disabling mailto links for sample email IDs in a documentation

### DISCUSSION THREAD

**Discussion by nanditha938** [facebook/docusaurus#10246] [Q&A]
In Docusaurus v3.4, all email IDs automatically appear as mailto links on the webpage.
 Is there a way to disable the mailto links for certain sample email IDs used in a documentation?

**Comment by slorber** [comment]
This is because we use [Remark-gfm](https://github.com/remarkjs/remark-gfm) by default, to have a similar behavior compared to what you see here on GitHub: seb@zy.com

We don't provide a way to disable this, and it's a behavior we always had afaik. It was probably already the case for Docusaurus v2.0.
